quickfixj-base/src/main/java/quickfix/UtcDateOnlyField.java

```diff
@@ -34,7 +35,11 @@ public UtcDateOnlyField(int field) {
     protected UtcDateOnlyField(int field, LocalDate data) {
         super(field, data);
     }
-        
+    
+    protected UtcDateOnlyField(int field, String data) {
```

Comment:
Is this correct? If I am not mistaken then ISO dates could also bear a time zone offset which UTC dates do not have.

Reply:
The doc is a bit confusing isn't it/
https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#BASIC_ISO_DATE

Here is the code from https://github.com/frohoff/jdk8u-dev-jdk/blob/master/src/share/classes/java/time/format/DateTimeFormatter.java

```
    public static final DateTimeFormatter BASIC_ISO_DATE;
    static {
        BASIC_ISO_DATE = new DateTimeFormatterBuilder()
                .parseCaseInsensitive()
                .appendValue(YEAR, 4)
                .appendValue(MONTH_OF_YEAR, 2)
                .appendValue(DAY_OF_MONTH, 2)
                .optionalStart()
                .appendOffset("+HHMMss", "Z")
                .toFormatter(ResolverStyle.STRICT, IsoChronology.INSTANCE);
    }
```
So it looks like the input could have TZ offset and then the date would not be UTC.  I suppose a custom formatter is required.